docs/Reply.md

```diff
@@ -109,6 +109,16 @@ fastify.get('/', async function (req, rep) {
 Sets a response header. If the value is omitted or undefined, it is coerced
 to `''`.
 
+`NOTE:`
```

Comment:
Instead of a note, I think it should be a sub-section dedicated to `set-cookie`.
It is helpful to explain the whole logic for how `set-cookie` will work differently from the others.
e.g. How to reset the `set-cookie`? What if I pass an array?

Reply:
Yeah this seems a better idea. So where should I place this `set-cookie` section in the reply documentation? Any specific place you want or maybe I can add it like a subsection after `reply.header` part?

Reply:
I think sub-section under `reply.header` is enough. And remember to update the `list of content` on the top of this file. It should be very clear to the reader there are something special of `set-cookie` when they start reading this readme.

Reply:
Like [this](https://github.com/genzyy/fastify/blob/fix-3409/docs/Reply.md#set-cookie) right?

Reply:
Done!